rj_gameplay/README.md

```diff
@@ -11,32 +11,44 @@ Stable code for gameplay.
 
 **Implementation - `rj_gameplay/`**
 
-Frequently changing code of gameplay .
+Frequently changing code of gameplay.
 
 **Tests - `tests/stp`**
 
 Unit tests for stp/.
 
 ## STP Framework
 
-Our gameplay library adapts the [STP
+Our gameplay library adopts the name of the [STP
 framework](https://citeseerx.ist.psu.edu/viewdoc/download?doi=10.1.1.61.1972&rep=rep1&type=pdf)
-proposed by CMDragons.
+proposed by CMDragons, but is quite different.
 
-**Skill - `rj_gameplay/skill/`**
+**Play - `rj_gameplay/play/`**
 
-Atomic robot behavior which consists of a behavior tree and calls a ROS action.
-(Examples: Move, Kick)
+Handles full team behavior. Gets potential roles from a list of >=6 tactics,
+then assigns 6 of them at any time.
+
+(Examples: Basic Defense)
 
 **Tactic - `rj_gameplay/tactic/`**
 
-Handles complex single robot behavior. Think of as one role in a play.
-(Examples: Goalie Receiver Waller)
+Handles role costing for one or more robots, such that Plays can choose
+higher-level ideas for each robot. Think of as abstraction layer between single-robot
+Skills and whole-team Plays.
```

Comment:
May want to mention that tactics typically are stateful or require some complex manipulation of worldstate info

Reply:
I don't love either detail because skills are also stateful and take in worldstate info. "Complex manipulation" is the only differentiation and that's hard to define. That's why I chose to write that they "handle role costing" because it defines better what the complex manipulation is. Do you have a better way to differentiate those descriptors from skills?

Reply:
Just my suggestions/thoughts: If you're just trying to better differentiate tactics and skills here, I think it would be useful to emphasize how skills handle the specific actions of the robot through the behavior tree while tactics handle redistributing/organizing those skills. Tactics, to my understanding, are mainly used to handle the assignment of robots to skills for a given play, and don't directly "tell" any specific robot what actions they should be performing at any moment (which skills do with the behavior tree). Tactics instead use the worldstate info to reorganize the necessary skills among the robots with new role requests. You mention the behavior tree below in the skills section, so I'm not sure if its necessary to repeat that info for clarification. If anything is unclear or my current understanding is flawed, feel free to DM me on Slack.